src/routes/etudiants/+page.ts

```diff
@@ -0,0 +1,15 @@
+import { error } from '@sveltejs/kit';
+import { supabase } from '$lib/auth';
+import type { PageLoad } from './$types';
+
+export const load = (async () => {
+	const { data: profile, error: err } = await supabase.from('Profile').select('*');
```

Comment:
`profile` should be `profiles` to easily distinct an Array of Profiles VS a single Profile